frontend/summary/summaryTable/studyEvaluationTable/Table.js

```diff
@@ -338,10 +339,10 @@ class EditColumnForm extends Component {
                                         : store.settings.columns[colIdx].width;
                             store.updateStagedColumn(colIdx, {width});
                         }}
-                        value={col.width}
-                        required={true}
```

Comment:
@casey1173 @rabstejnek it looks like the only change is removing the required attribute. I guess if the width is unspecified, is there any way to assume the width is 1 for the SET table?

Reply:
it only updates the table width if the input is valid, plus when you create a table the initial width value is 1. so if a user removes the width val and presses update, the width will stay 1 (or whatever it was before)